src/Symfony/Component/HttpKernel/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+4.2.0
+-----
+
+ * deprecated `KernelInterface::getName()` and the `kernel.name` parameter
```

Comment:
Should we mention the alternatives available? `KernelInterface::getContainerClass()` and `kernel.container_class`

Reply:
No, there are no alternative. Apps do not need a Kernel name. People can use the Kernel class name if they want to, but I don't see any use cases.